src/Workspaces/Core/Portable/FindSymbols/SymbolTree/SymbolTreeInfo.cs

```diff
@@ -95,6 +90,26 @@ public IEnumerable<ISymbol> Find(IAssemblySymbol assembly, Func<string, bool> pr
             }
         }
 
+        /// <summary>
+        /// Slow, linear scan of all the symbols in this assembly to look for matches.
+        /// </summary>
+        public IEnumerable<ISymbol> Find(IAssemblySymbol assembly, Func<string, bool> predicate, CancellationToken cancellationToken)
+        {
+            for (int i = 0, n = _nodes.Count; i < n; i++)
```

Comment:
Right, but does that optimization matter? If so, a //PERF comment would keep someone from undoing it in the future.